components/brave_wallet/common/brave_wallet.mojom

```diff
@@ -1879,6 +1880,8 @@ interface SolanaTxManagerProxy {
 
   GetEstimatedTxFee(string chain_id, string tx_meta_id)
     => (uint64 fee, SolanaProviderError error, string error_message);
+  GetSolanaGasEstimation(string chain_id, string tx_meta_id)
+    => (SolanaGasEstimation estimation, SolanaProviderError error, string error_message);
```

Comment:
Why do we need two instead of just changing fee to the new struct in GetEstimatedTxFee?